Docs: debugging.rst reworked (#6592)
WeblateOrg/weblate: docs/contributing/debugging.rst

Not processing background tasks
-------------------------------

> A lot of things happen in background with Celery workers.

Reviewer: ```suggestion
A lot of things happen in the background Celery workers.
```

Author: ```suggestion
A lot of things are done in the background by Celery workers.
```
